pkg/util/cgroup/cgroup_cpu_linux.go

```diff
@@ -57,6 +57,11 @@ func CPUQuotaToGOMAXPROCS(minValue int) (int, CPUQuotaStatus, error) {
 	return maxProcs, CPUQuotaUsed, nil
 }
 
+// GetCPUPeriodAndQuota returns CPU period and quota time of cgroup.
+func GetCPUPeriodAndQuota() (int64, int64, error) {
```

Comment:
```getCgroupCPU``` has got the ```Period```, ```Quota``` of CPU.

Reply:
> `getCgroupCPU` has got the `Period`, `Quota` of CPU.

Yes it does. But we only want to get period and quota here. Don't want to call getCgroupCPU to make additional calls to detectCPUUsageInV1/detectCPUUsageInV2.